.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,66 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+	"name": "Mobile Wallet Test Dev",
+	//"workspaceMount": "source=${localWorkspaceFolder}/aries-mobile-tests,target=/aries-mobile-tests,type=bind",
+	//"workspaceFolder": "/aries-mobile-tests",
+	// Use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"build": {
+        // Path is relative to the devcontainer.json file.
+		"context": "../aries-mobile-tests",
+        "dockerfile": "../aries-mobile-tests/Dockerfile.dev-harness"
+    },
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+		"docker-from-docker": "latest"
+	},
+
+	// If using a local android device/emulator you will need to mount to the location of the app apk so that it can be installed.
+	// Change this to your absolute path to the location of the android apk you want to test with
+	"mounts": [
+		{ "source": "/Users/Shel/Projects/BC.gov/apps/bc-wallet", "target": "/bc-wallet/app", "type": "bind" },
```

Comment:
not sure about absolute paths. Any creative ways to address it? I think of a dev container something that works out of the box to make dev life easier. The need to update paths might create false changes. But, I am okay with this being a first pass.

Reply:
What is the purpose of that path? Just a place to get the app APK file?

Reply:
Yes.

Reply:
`${localWorkspaceFolder}/apps/bc-wallet`?

Reply:
Could do that but I don't think it saves any effort.  /apps/bc-wallet/ isn't in the repo and you will have to create it and put the apk there anyway, so why not just leave it open to put the app where ever you want?